workspaces/libnpmpublish/lib/provenance.js

```diff
@@ -193,7 +193,8 @@ const generateProvenance = async (subject, opts) => {
       },
     }
   }
-  return sigstore.attest(Buffer.from(JSON.stringify(payload)), INTOTO_PAYLOAD_TYPE, opts)
+  return sigstore.attest(Buffer.from(JSON.stringify(payload)), INTOTO_PAYLOAD_TYPE,
+    { ...opts, legacyCompatibility: true })
```

Comment:
When updating to `sigstore@3.0.0` the `legacyCompatibility` flag is necessary to ensure that we continue to generate attestations which are verifiable by older versions of npm.

Reply:
After more research we don't need to enable this.  npm 10.6.0 and beyond can verify these new attestations, and Node.js 18 currently ships with npm 10.7.0.  So all supported Node.js/npm versions can verify the new format.